docs/wiki/DEX-Trading-Account-Management.md

```diff
@@ -0,0 +1,104 @@
+# Guide to Managing Your DEX Trading Account
+
+In this guide, we will discuss how you can manage your DEX Trading Account.
+
+Let's get started!
+
+## Setting Up Your DEX Trading Account
+
+Before you can start buying and selling on Decred DEX, you'll need a DEX trading
+account. If you've already followed all the steps in the [Initial Setup
+Guide](./Client-Installation-and-Configuration.md/#initial-setup), you're all
+set. If not, let's create your DEX trading account together:
+
+1. Go to the `Markets` Page.
+2. Find the `Create Account` button and click it.
+
+    <img src="./images/create-account-market-page.png" width="300">
+
+3. Follow the steps in the [Initial Setup Guide](./Client-Installation-and-Configuration.md/#initial-setup) from step 4.
+
+**IF** you've already created the wallet you plan to use and funded it with the
+required bond amount, you can skip steps 6 and 7 of the [Initial Setup
+Guide](./Client-Installation-and-Configuration.md/#initial-setup).
+
+## Adding an existing DEX Trading Account
+
+If you have an existing DEX account that you want to use with a new setup, here's how you can do it:
+
+1. Go to the `Settings` Page.
+2. Click on `Import Account`.
+
+    <img src="./images/dex-accounts-settings.png" width="300">
+
+3. Choose the exported DEX account `.json` file by clicking on `load from file`.
```

Comment:
Have input instructions but no corresponding export instructions. Looking at the dex settings page I think it's obvious but maybe a note here or should that be another `##` point?

Reply:
Thanks for the review @JoeGruffins. I'll include a guide on how to export.